dbms/src/test/java/org/polypheny/db/sql/view/ComplexViewTest.java

```diff
@@ -2086,7 +1870,7 @@ public void testQ20() throws SQLException {
 
     // SELECT NOT POSSIBLE
     // java.lang.AssertionError: type mismatch: ref: INTEGER NOT NULL input: VARCHAR(1) NOT NULL
-    @Ignore
+    @Disabled
```

Comment:
Can you enable this test?

Reply:
Yes, after https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1519992631, enabled.